src/test/java/trackr/model/ModelManagerTest.java

```diff
@@ -114,19 +142,25 @@ public void equals() {
         assertFalse(modelManager.equals(5));
 
         // different addressBook -> returns false
-        assertFalse(modelManager.equals(new ModelManager(differentAddressBook, userPrefs)));
+        assertFalse(modelManager.equals(new ModelManager(differentAddressBook, taskList, userPrefs)));
+
+        // different taskList -> returns false
+        assertFalse(modelManager.equals(new ModelManager(addressBook, differentTaskList, userPrefs)));
 
         // different filteredList -> returns false
-        String[] keywords = ALICE.getName().fullName.split("\\s+");
-        modelManager.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(keywords)));
-        assertFalse(modelManager.equals(new ModelManager(addressBook, userPrefs)));
+        String[] personKeywords = ALICE.getName().fullName.split("\\s+");
+        modelManager.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(personKeywords)));
+        String[] taskKeywords = SORT_INVENTORY_N.getTaskName().fullTaskName.split("\\s+");
+        modelManager.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(taskKeywords)));
+        assertFalse(modelManager.equals(new ModelManager(addressBook, taskList, userPrefs)));
```

Comment:
perhaps we can separate this test case into 2 different test case, one testing a different filteredPersonList, while one test a different filteredTaskList?